src/index.html

```diff
@@ -22,6 +22,357 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo-container">
+        <a
+          class="header__logo"
+          href="#"
+        >
+          <img
+            src="./images/logo.png"
+            alt="logo"
+          />
+        </a>
+      </div>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__list-item">
+            <a
+              href="#"
+              class="header__link header__link--is-active is-active"
```

Comment:
The class `is-active` is redundant here since `header__link--is-active` is already being used to indicate the active state of the link. You should use only one class to style the active state according to the BEM methodology.